codex/blocktype.nim

```diff
@@ -35,7 +35,7 @@ type
     cid*: Cid
     data*: seq[byte]
 
-template EmptyCid*: untyped =
+template emptyCid*: untyped =
```

Comment:
Hmm, this is weird, but we're kinda faking types here, so it would be convenient to leave it capitalized. Any way we could ignore some symbols like this?

Reply:
I agree. Will apply Tanguy's suggestion.